ns/activitystreams.jsonld

```diff
@@ -370,6 +370,10 @@
     "shares": {
       "@id": "as:shares",
       "@type": "@id"
+    },
+    "alsoKnownAs": {
+      "@id": "as:alsoKnownAs",
+      "@type": "@id"
```

Comment:
This seems like the right decision, making it `@type: @id`.  I assume then that the DID spec will, in its own json-ld context, mark it as a list, but since in AP applications it's only assumed to have a single value (I think?  confirmation @nightpool?) this makes sense to me for the generalized AS2 json-ld context.

Reply:
Wellll unless I'm mixing everything up, in RDF land there is no list, just several triples with the same subject and predicate and various different objects. This is communicated as a list in JSON-LD syntax for the *data*, but for the *context* `@type: @id` here sets the datatype for each item in the list. Is that right @msporny?

Reply:
Agree that there's no "list" here, but in JSON-LD land it might look like a list, because JSON-LD uses JSON arrays to specify multiple object with the same relationship to the subject (the document). I don't know specifically whether Mastodon expects it to have a single value or not, but I don't see any reason to restrict it to a single object, so I have it written up in my definition as `functional: false` to indicate that.

From a JSON-LD perspective, this makes sense, unless we do think the order is important, in which case we should give it a `@list` type. My understanding is that consensus is that order is unimportant though!

Reply:
@nightpool Right! exactly. can't see why order would be important.

Reply:
What I really meant here is the absence of `"@container": "@list"` :)